Gemfile

```diff
@@ -58,7 +58,7 @@ gem 'druid-tools'
 gem 'dry-types'
 gem 'edtf'
 gem 'faraday', '~> 2.0'
-gem 'globus_client', '~> 0.2'
+gem 'globus_client', '~> 0.3'
 gem 'honeybadger', '~> 4.0'
```

Comment:
we need at least 0.3 to ensure we have access to some new methods for checking user existence